content/learning-paths/smartphones-and-mobile/ray_tracing/rt04_acceleration_structure.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Acceleration structure"
+weight: 5
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Acceleration structure
+
+For ray tracing, the first thing that we need is to create an acceleration structure to represent our scene.
+
+An acceleration structure is an optimised data structure for fast ray intersection tests. It organizes the geometry so that we can efficiently narrow down what each ray intersects. Vulkan represents them using `VK_KHR_acceleration_structure`, and although the actual structure is opaque and defined by the implementation, it is usually implemented as a treelike structure, and the API gives us some options to control the topology and keep it balanced.
+
+The API makes a distinction between *Bottom-Level Acceleration Structures* (BLAS) and *Top-Level Accelerations Structures* (TLAS).
+
+*Bottom-Level Acceleration Structures* contain the actual geometry data, usually as triangles, but they could also be mathematically defined models. The actual implementation of a BLAS is opaque to users, but the driver usually organizes this data in a Bounding Volume Hierarchy (BVH), so that when we cast a ray, we can minimize the number of triangle intersection checks.
+
+*Top-Level Accelerations Structures* contain other Bottom-Level Acceleration Structures. Top-Level Acceleration Structures use instances to group BLASes, and link them with other properties, such as a custom ID, which we can use to define a material; or a transform matrix, which we can use to efficiently update animations.
```

Comment:
"TLASs contain other BLASs" - shouldn't be an "other" there. Paragraph's second sentence is also quite long/complicated - worth splitting in two?